BlackNectar/BlackNectar/CustomButtonView.swift

```diff
@@ -30,48 +30,55 @@ import UIKit
    
     @IBInspectable var circular: Bool = false {
         
-        didSet
-        {
+        didSet {
             updateView()
         }
     }
     
     @IBInspectable var borderWidth: CGFloat = 1 {
         
-        didSet
-        {
+        didSet {
             updateView()
         }
     }
     
     @IBInspectable var borderColor: UIColor = UIColor.white {
         
-        didSet
-        {
+        didSet {
             updateView()
         }
     }
     
     @IBInspectable var shouldRasterize: Bool = false {
         
-        didSet
-        {
+        didSet {
             updateView()
         }
     }
     
     @IBInspectable var shadowOffSet: CGSize = CGSize(width: 3, height: 0) {
         
-        didSet
-        {
+        didSet {
             updateView()
         }
     }
     
     @IBInspectable var cornerRadius: CGFloat = 5 {
         
-        didSet
-        {
+        didSet {
+            updateView()
+        }
+    }
+    
+    @IBInspectable var numberOfLines: Int = 1 {
+        
+        didSet {
+            
+            guard numberOfLines > 0 else {
```

Comment:
Even if the `numberOfLines` fails this guard, `updateView()` should still be called.